examples/puppetmaster/smoketest_e2e.sh

```diff
@@ -62,6 +68,11 @@ wait_for_puppetmaster() {
   echo "OK"
 }
 
+install_required_module_dependency() {
+  echo "Installing puppetlabs-registry module dep to server..."
+  docker-compose exec -T puppet puppet module install puppetlabs-registry
```

Comment:
why is this required?

Reply:
some (eg. windows) use of our plugin fails otherwise. it's a dependency that isn't automatically resolved given the way we add our plugin directly.

Reply:
is this something that needs to be documented in the README (eg for windows users)? can this PR include more context for why this change was added?

Reply:
When installing from puppet forge, this isn't needed as the dependency should be automatically resolved since we have it in the [explicit dependency list](https://github.com/cyberark/conjur-puppet/blob/master/metadata.json#L11).